components/renderers/widgets/widgetb/widgetb-renderer.ts

```diff
@@ -0,0 +1,64 @@
+///<reference path="../../../references.ts"/>
+
+class WidgetBRenderer implements JSONForms.IRenderer {
+
+    private maxSize = 99;
+    priority = 99;
+
+    constructor(private renderService: JSONForms.IRenderService, private pathResolver: JSONForms.IPathResolver) { }
+
+    render(element: IControlObject, subSchema: SchemaArray, schemaPath: string, services: JSONForms.Services): JSONForms.IRenderDescription {
+        var data = services.get<JSONForms.IDataProvider>(JSONForms.ServiceId.DataProvider).getData();
+
+        if (!Array.isArray(data)) {
+            data = this.pathResolver.resolveInstance(data, schemaPath);
+        }
+
+        var dataString = JSON.stringify(data);
```

Comment:
WidgetBRenderer is the one showing controls in different groups. WidgetARenderer has lower priority so it's never applied. I think is better not including widgeta by the moment.